server/src/instant/dash/routes.clj

```diff
@@ -285,6 +288,27 @@
                                                                                 500 400))))]
     (response/ok {:metrics metrics-with-b64-charts})))
 
+(defn admin-overview-daily-get [req]
+  (let [{:keys [email]} (req->auth-user! req)
+        _ (assert-admin-email! email)
+        conn (aurora/conn-pool :read)
+        target-date (.minusDays (LocalDate/now) 1)
```

Comment:
If you use `Instant` instead of `LocalDate`, then you'll probably get better consistency between prod and dev.